src/main/kotlin/com/terraformation/backend/tracking/db/ObservationResultsStore.kt

```diff
@@ -684,7 +685,17 @@ class ObservationResultsStore(private val dslContext: DSLContext) {
 
               val mortalityRate = species.calculateMortalityRate()
               val mortalityRateStdDev =
-                  monitoringPlots.mapNotNull { it.mortalityRate }.calculateStandardDeviation()
+                  monitoringPlots
+                      .mapNotNull { plot ->
+                        plot.mortalityRate?.let {
```

Comment:
This would be slightly clearer with a named lambda variable so the last line is more like `mortalityRate to permanentPlots`.